packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/AnnotateButton/AnnotateButton.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import AnnotateButton from './AnnotateButton'
+
+const args = {
+  dark: false,
+  active: false,
+  onClick: () => console.log('clicked')
```

Comment:
If you use actions in these stories, each click will show a message in the Actions panel. 
https://storybook.js.org/docs/react/essentials/actions/

Reply:
Oh true! I'll add them as actions instead.